pipeline/src/scripts/documentIds.ts

```diff
@@ -1,7 +1,7 @@
 import { AddressablesAllowedTypes } from '@weco/content-pipeline/src/helpers/getGraphQuery';
 
 const documentIds: { [key: string]: string | undefined } = {
-  article: 'ZdSMbREAACQA3j30',
+  article: 'aAi4nhEAACMAfdxX',
```

Comment:
We'd discussed adding a test to make sure the article in here always had featured links (could be separate work), still has value?